spec/support/features/in_person_helper.rb

```diff
@@ -42,7 +43,7 @@ def fill_out_state_id_form_ok(double_address_verification: false, same_address_a
            from: t('in_person_proofing.form.state_id.state_id_jurisdiction')
     fill_in t('in_person_proofing.form.state_id.state_id_number'), with: GOOD_STATE_ID_NUMBER
 
-    if double_address_verification
+    if double_address_verification && capture_secondary_id_enabled
```

Comment:
good point, i did think about removing `double_address_verification` from the places where I checked `capture_secondary_id_enabled` but had decided against it since other tests are still using dav and wanted to minimize any confusion. But it is a bit redundant. I'll remove `dav` from the tests I updated.